### ISSUE TITLE
Tag all publications with MaRDI profile type

### ISSUE THREAD

**Issue by eloiferrer** [MaRDI4NFDI/portal-compose#509]
**Issue description**:
So far we have only tagged the items that have a zbMATH DE number.

This does not include publications that are imported from Crossref. To tag those I suggest to tag all the items that are an 'instance of':
- Scholarly article: https://portal.mardi4nfdi.de/wiki/Item:Q56887
- Report: https://portal.mardi4nfdi.de/wiki/Item:Q57006
- Book: https://portal.mardi4nfdi.de/wiki/Item:Q56401
- Book series: https://portal.mardi4nfdi.de/wiki/Item:Q56818
- Book set: https://portal.mardi4nfdi.de/wiki/Item:Q57064
- Book section: https://portal.mardi4nfdi.de/wiki/Item:Q56939
- Dissertation: https://portal.mardi4nfdi.de/wiki/Item:Q56927

In other words, we consider a publication to be any item that is an instance of one of entities listed above. 
 
**TODOS**: 

- [ ] example todo to copy 

**Acceptance-Criteria** 

**Checklist for this issue**: 

- [ ] Assignee has been set for this issue 
- [ ] All fields of the issue have been filled 
- [ ] Issue has been assigned to the main project
- [ ] Code was merged
- [ ] Feature branch has been deleted and issues were updated / closed
- [ ] Issue is tracked by an epic, or the label 'non-epic' is set to the issue.

**Comment by eloiferrer** [comment]
To find scholarly articles with no zbMATH DE number:
SELECT ?item WHERE { 
  ?item wdt:P31 wd:Q56887.
  FILTER NOT EXISTS { ?item wdt:P1460 wd:Q5976449. }
}
